src/test/compile-fail/lint-type-overflow2.rs

```diff
@@ -9,16 +9,18 @@
 // except according to those terms.
 //
 
-#![deny(overflowing_literals)]
-#![deny(const_err)]
+#![warn(overflowing_literals)]
```

Comment:
I see a few tests changed from `deny(lint)` to `warn(lint)`, did they start failing with `deny`?